docs/general/ddl/data-types.rst

```diff
@@ -234,9 +234,9 @@ are likely to be larger due to additional metadata.
       - variable
       - Each coordinate is stored as a ``DOUBLE PRECISION`` type.
       - A ``GEO_SHAPE`` column can store different kinds of `GeoJSON geometry objects`_.
-    * - ``FLOAT_VECTOR``
-      - variable
-      - Minimum length: 1. Maximum length: 2^31-1
+    * - ``FLOAT_VECTOR(n)``
+      - ``n``
+      - Vector Minimum length: 1. Maximum length: 1024.
```

Comment:
maybe worth mentioning that the default length if no dims are specified is `1`?  or if not here later on in the detailed description as a note ?

Reply:
I'd actually tend to change it into an error if the length is missing. Using length=1 is probably a rare case and not warranted to default to it. Leaving it unspecified could be an oversight.

Reply:
:+1: